.github/pull_request_template.md

```diff
@@ -1,10 +1,8 @@
 ## PR pre-checks (self review)
 <!--- PR should be created as Draft, when CI finished and relevant checkboxes selected, add reviewers and then click on "Ready for review" button.-->
 <!--- Put an `x` in all the boxes that apply or create PR and then click on all relevant checkboxes: -->
-- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
-- [ ] I didn't leave commented-out/debugging code
 - [ ] I added the relevant `backport` labels
+- [ ] I have added unit tests to cover my changes (under `unit-test/` folder)
 - [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
```

Comment:
I would remove this one as well, and possibly add a "reminder" section, that it can be listed there.

Reply:
moved